src/main/java/com/softserve/ita/java442/cityDonut/model/ProjectStatus.java

```diff
@@ -0,0 +1,26 @@
+package com.softserve.ita.java442.cityDonut.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="project_status")
+public class ProjectStatus {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @NotNull
```

Comment:
remove @ NotNull...add @Column(name = "status", nullable = false)...default length 255, you need length less than 255, and add length to column where it`s need